htlcswitch/switch.go

```diff
@@ -1717,6 +1738,28 @@ out:
 			totalSatSent += diffSatSent
 			totalSatRecv += diffSatRecv
 
+		// The ack ticker has fired so if we have any settle/fail entries
+		// for a forwarding package to ack, we will do so here in a batch
+		// db call.
+		case <-s.cfg.AckEventTicker.Ticks():
+			// If the current set is empty, pause the ticker.
+			if len(s.pendingSettleFails) == 0 {
```

Comment:
had considered this, but as is this will prevent tearing down and reinitializing the ticker under load, since o/w we would expect to tear it down after every tick